services/processor/native/adapter/native_compiler.go

```diff
@@ -81,7 +82,7 @@ func (c *nativeCompiler) warmUpCompilationCache() {
 	}
 }
 
-func (c *nativeCompiler) Compile(ctx context.Context, code string) (*sdkContext.ContractInfo, error) {
+func (c *nativeCompiler) Compile(ctx context.Context, code ...string) (*sdkContext.ContractInfo, error) {
 	c.metrics.sourceSize.Record(int64(len(code)))
```

Comment:
question ... since you rely on order of files and then name them later with .i ... are we 100% all nodes read the files the same order ?

Reply:
Since they use the same compiler, I think yes. The source code is the same in the end.